python/cudf/cudf/_lib/interop.pyx

```diff
@@ -54,35 +27,13 @@ def to_dlpack(list source_columns):
     """
     if any(column.null_count for column in source_columns):
         raise ValueError(
-            "Cannot create a DLPack tensor with null values. \
-                Input is required to have null count as zero."
-        )
-
-    cdef DLManagedTensor *dlpack_tensor
-    cdef table_view source_table_view = table_view_from_columns(source_columns)
-
-    with nogil:
-        dlpack_tensor = cpp_to_dlpack(
-            source_table_view
+            "Cannot create a DLPack tensor with null values. "
+            "Input is required to have null count as zero."
```

Comment:
Can we move this check into the pylibcudf function? The same restrictions apply there.